app/models/page.rb

```diff
@@ -94,6 +94,10 @@ def title
     agentize_title(contents.match(/^\#\s(.+)/).try(:[], 1) || "")
   end
 
+  def description
+    extracted_data.fetch("shortDescription")
```

Comment:
This will raise if a short description doesn't exist, but by design. The tests make sure that every page renders, which means every page has a description. And the data extractor should be able to make a description for every page as long as it has even a little bit of content, and no page should have no content.